### PR TITLE
[#18] Add templates and ci check

### PR DESCRIPTION
**Pull request by worm2fed**
Problem:
We do not have templates for PR and issues, also there are no ci checks.
So, someone can forgot to do all required actions before merge or can
frustrate with thoughts how to describe an issue.

Solution:
1. Add templates for issues and PR
2. Add ci check for licenses. This is first step to later add more checks.

Resolves: #18

### CODE REVIEW

**Comment by dcastro** [comment]
I created an issue so we can track this work here: #18

Please add the `[#18]: ` prefix to your commit messages.

Also, as discussed on Slack, please add the usual checks (build, test, hlint, etc) to the CI script.

**Comment by worm2fed** [comment]
@dcastro review it, please

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    commands:
+      - nix-build -A weeder-script
+      - ./result
+
+  # don't run tests until all components are built
+  - wait
+
+  - label: tests
+    commands:
+      - nix-build -A checks.x86_64-linux.test
```
Comment (dcastro) [inline .buildkite/pipeline.yml:38]: Is this actually running the tests or just building them?

I would have expected to see the test names in the logs, but there's nothing there: https://buildkite.com/serokell/haskell-algorand-sdk/builds/22#8f0e2142-0f2c-4508-a7b7-5b66ad70e6ec

Comment (worm2fed) [reply .buildkite/pipeline.yml:38]: hm....
https://buildkite.com/serokell/haskell-algorand-sdk/builds/14#7d983bf3-2ead-4f22-9402-177162c8f2b4

here we can see test names

Comment (worm2fed) [reply .buildkite/pipeline.yml:38]: @balsoft we cannot see tests logs at last CI launch because of caching?

Comment (balsoft) [reply .buildkite/pipeline.yml:38]: Yes, that's correct. Nix doesn't do stuff it has already done once.

File: .buildkite/pipeline.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    commands:
+      - nix run -f. legacyPackages.x86_64-linux.haskellPackages.hlint
+        -c hlint -j --hint .hlint.yaml */
+
+  - label: weeder
+    commands:
+      - nix-build -A weeder-script
+      - ./result
+
+  # don't run tests until all components are built
```
Comment (balsoft) [inline .buildkite/pipeline.yml:33]: No real reason to do this, nix can handle building&running tests together with the rest of the checks

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    is one) of each source file in the repository (yes, each and every source
+    file – it is not as hard as it might sound):
+
+    ```haskell
+    {- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+     -
+     - SPDX-License-Identifier: MPL-2.0
+     -}
+    ```
+
```
Comment (balsoft) [inline .github/CONTRIBUTING.md:37]: Not sure if this is copypasted from somewhere, but it should also recommend using `reuse addheader` which automates the process

**Review by balsoft — APPROVED** [review]
Looks alright

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (worm2fed) [reply .github/CONTRIBUTING.md:37]: oh! didn't know that

**Pull request merged**